rust/repositories.bzl

```diff
@@ -1,8 +1,36 @@
 load(":known_shas.bzl", "FILE_KEY_TO_SHA")
-load(":triple_mappings.bzl", "triple_to_system", "triple_to_constraint_set", "system_to_binary_ext", "system_to_dylib_ext", "system_to_staticlib_ext")
+load(":triple_mappings.bzl", "system_to_binary_ext", "system_to_dylib_ext", "system_to_staticlib_ext", "triple_to_constraint_set", "triple_to_system")
 
 DEFAULT_TOOLCHAIN_NAME_PREFIX = "toolchain_for"
 
+def rust_repositories():
```

Comment:
Took me a second to find current version; feel free to veto the move.

Reply:
No worries, change LGTM